onecgiar-pr-server/src/api/results/entities/result.entity.ts

```diff
@@ -420,4 +393,7 @@ export class Result {
 
   @OneToMany(() => Notification, (ra) => ra.obj_result)
   obj_result_notification: Notification[];
+
+  @OneToMany(() => ResultQaedLog, (ra) => ra.obj_result_id_qaed)
+  obj_result_qaed: Notification[];
```

Comment:
@JuanGuzman-io i don't think `Notification` is the correct type

Reply:
@gcmartinezg thank you for noticing this error.